.github/workflows/push-release.yml

```diff
@@ -0,0 +1,477 @@
+      - name: Load Secrets from Vault
+        uses: rancher-eio/read-vault-secrets@main
+        with:
+          secrets: |
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.DOCKER_USERNAME }}
+          password: ${{ env.DOCKER_PASSWORD }}
+      - name: Create manifest list and push
+        run: |
+          docker buildx imagetools create -t ${{ env.IMAGE_INSTALLER }}:${{ env.TAG }} ${{ env.IMAGE_INSTALLER }}:${{ env.TAG }}-amd64 ${{ env.IMAGE_INSTALLER }}:${{ env.TAG }}-arm64
+          if [[ "${{ github.ref_name }}" == release/v* ]]; then
+            docker buildx imagetools create -t ${{ env.IMAGE_INSTALLER }}:${{ env.HEAD_TAG }} ${{ env.IMAGE_INSTALLER }}:${{ env.TAG }}-amd64 ${{ env.IMAGE_INSTALLER }}:${{ env.TAG }}-arm64
+          fi
+      - name: Inspect image
+        run: |
+          docker buildx imagetools inspect ${{ env.IMAGE_INSTALLER }}:${{ env.TAG }}
+  create-images-files:
+    if: github.event_name == 'push' && contains(github.ref, 'tags/') # Only run on push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Environment Variables
+        uses: ./.github/actions/setup-tag-env
+      - name: Download data.json
+        run: |
+          mkdir -p bin
+          curl -sLf https://releases.rancher.com/kontainer-driver-metadata/${{ env.CATTLE_KDM_BRANCH }}/data.json > ./bin/data.json
+      - name: Create files
+        run: |
+          mkdir -p $HOME/bin
+          touch $HOME/bin/rancher-rke-k8s-versions.txt
+      - name: Create components and images files
+        run: ./scripts/create-components-images-files.sh
+      - name: Move files
+        run: |
+          mv $HOME/bin/* ./dist
+          mv ./bin/*.txt ./dist
+          mv ./bin/*.sh ./dist
+          mv ./bin/*.ps1 ./dist
+      - name: Create sha256sum.txt file
+        run: ./scripts/artifacts-hashes.sh
+      - name: Upload artifacts
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh release upload -R ${{ env.REPOSITORY_OWNER }}/rancher ${{ env.TAG }} ./dist/* --clobber
+  docker-image-digests:
+    if: github.event_name == 'push' && contains(github.ref, 'tags/') # Only run on push
+    runs-on: ubuntu-latest
+    needs: [create-images-files]
+    env:
+      DOCKER_REGISTRY: "docker.io"
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Environment Variables
+        uses: ./.github/actions/setup-tag-env
+      - name: Create dist folder
+        run: mkdir -p dist
+      - name: Setup ecm-distro-tools
+        uses: rancher/ecm-distro-tools@v0.38.1
+        with:
+          version: v0.38.1
```

Comment:
Just to understand if this version is also needed.

Reply:
Yes, `@v0.38.1` is the action version, and `version: v0.38.1` is the actual ecm distro tools binary version, sometimes the action is updated, but the binary isn't updated immediately.

Reply:
Thanks for the explanation. Marking this as resolved.